libraries/APM_OBC/APM_OBC.cpp

```diff
@@ -117,6 +117,30 @@ const AP_Param::GroupInfo APM_OBC::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("MAX_COM_LOSS", 14, APM_OBC, _max_comms_loss, 0),
 
+    // @Param: GEOFENCE
+    // @DisplayName: Enable geofence Advanced Failsafe
+    // @Description: This enables the geofence part of the AFS. Will only be in effect if AFS_ENABLE is also 1
+    // @User: Advanced
+    AP_GROUPINFO("GEOFENCE",     15, APM_OBC, _enable_geofence_fs, 1),
```

Comment:
yeah, I'm just looking though it more closely and I realize that teh logic is backwards on them, 1 to keep current behavior. I should talk to @tridge on this one, there are two issues: teh backwards logic (1 to keep existing behavior, 0 to change it) and changing a param from 16bit to 32bit which will corrupt the EEPROM.